.circleci/RoboFile.php

```diff
@@ -410,7 +408,7 @@ public function configureModuleDependencies()
     $config = json_decode(file_get_contents('composer.json'));
 
     // The Drupal core image might need updating. Request the newest stable.
-    $config->require->{"drupal/core"} = "~8.8";
+    unset($config->require->{"drupal/core"});
```

Comment:
The next line is `$config->require->{"drupal/core-recommended"} = "~8.8";`. We require `drupal/core-recommended` instead of `drupal/core` because _core-recommended_ locks down core dependencies correctly, and makes sure we are testing against the same library  versions core has been tested against (see https://www.drupal.org/docs/8/update/update-core-via-composer#migrate-webflo-drupal-core-strict-to-core-recommended)